src/views/GScan.vue

```diff
@@ -57,6 +57,15 @@
                   mdi-table-edit
                 </v-icon>
               </td>
+              <td class="justify-center">
+                <v-icon
+                    small
+                    class="mr-2"
+                    @click="viewGraph(item)"
+                >
+                  mdi-vector-polyline
+                </v-icon>
+              </td>
```

Comment:
Need to add another column to the `headers` variable.

Reply:
I thought this would be part of Actions?

Reply:
If so, then the `<v-icon>` element should live with the previous `<td>`?

Reply:
which do you prefer?

Reply:
Don't mind. Probably easiest for them to live under the same `<td>`?

Reply:
It'll all change later so whatever works for now...